content/reference/user/private_locations/index.md

```diff
@@ -0,0 +1,104 @@
+lead: "Private Locations on your AWS account"
+date: 2021-11-07T14:29:04+00:00
+lastmod: 2021-11-07T14:29:04+00:00
+weight: 21070
+---
+
+A location describe where to start load generators:
+- Public locations are available by default, and listed by regions.
+- Private locations must be configured, and allow you to deploy load generators in your infrastructure.
+
+{{< alert tip >}}
+Private locations allow you to load test in a private infrastructure, without any inbound connection or credential sharing.
+To access this feature, please contact our [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59?summary=Private+Locations&description=Contact%20email%3A%20%3Cemail%3E%0A%0AHello%2C%20we%20would%20like%20to%20enable%20the%20private%20locations%20feature%20on%20our%20organization.).
+{{< /alert >}}
+
+## Introduction
+
+To configure private location, you must configure an agent in your infrastructure, called a **Control Plane**.
+
+This **Control Plane** will be in charge of spawning load generators on different locations, based on the configuration you'll provide.
+
+**TO BE UPDATED!**
+{{< img src="schema.jpg" alt="Infrastructure schema" >}}
+
+The **Control Plane** will pull periodically from our API gateway to know if there's any run to handle that requires one (or more) locations of this **Control Plane**.
+
+If so, it will start instances (based on the locations configurations) and start the simulation run on them. 
+Those instances will send stats through the API gateway as well.
+
+{{< alert info >}}
+Only two outbounds domain must be allowed in your network:
+- The API Gateway: `api.gatling.io` on port `443`
+- Download of the Gatling simulation on S3: `eu-west-3.amazonaws.com` on port `443`
+{{< /alert >}}
+
+## Control plane configuration
+
+### Control plane token
+
+Access the private locations section by clicking on the Private locations in the navigation bar (only visible if the feature is activated on your organization).
+
+Click on create **Control Plane**, and fill an unique identifier. (only lowercase, separated by underscores)
+
+{{< img src="create-control-plane.png" alt="Create control plane" >}}
+
+On the next modal, make sure to copy the **Control Plane** token, you’ll need it later.
+
+{{< img src="copy-token.png" alt="Copy control plane token" >}}
+
+You can see the **Control Plane** you just created under the Control planes tab.
+
+{{< img src="control-planes-uninitialized.png" alt="Control Planes table" >}}
+
+It's defined by an unique identifier, a description (provided later on by its configuration) and a status.
+
+Status have three possible value:
+- **Uninitialized:** Control plane never reached the application.
+- **Up:** Control plane is properly configured, uploaded its private locations configuration, and reaches periodically the application to fetch new runs.
+- **Down:** Control plane has been up, but didn't reach the application for a while.
```

Comment:
```suggestion
- **Uninitialized:** the control plane has never contacted the application
- **Up:** The control plane is properly configured, has uploaded the names of its configured private locations, and periodically calls the application to fetch new runs.
- **Down:** The control plane has been up, but hasn't called the application for a while.
```
grammar: present perfect rather than past simple, because we are dealing with the present consequences of a past action (helpful table: https://www.perfect-english-grammar.com/present-perfect-or-past-simple.html).

I think it's confusing to say that the control plane "uploaded its private locations configuration" - kinda implies it would;upload their AWS configuration...